tools/ansible/tasks/wsu/main.yaml

```diff
@@ -109,8 +111,18 @@
       retries: 3
       delay: 5
 
+    # Check if worker label is already applied to the Windows nodes bootstrapped via WMCB
+    # We should always get a single node that we've bootstrapped recently, if not skip this
+    # step
+    - name: Check if worker label is already applied to the Windows nodes bootstrapped via WMCB
+      delegate_to: localhost
+      shell: "oc get node {{ node_name.stdout_lines[0] }} -o yaml | grep -i node-role.kubernetes.io/worker"
```

Comment:
Why `node_name.stdout_lines[0]` despite of `when: node_name.stdout_lines | length == 1`?

Reply:
That's how we can access the first and only element as stdout_lines is a list

Reply:
How else would we access the single and only element in the list?